redaxo/src/addons/install/lib/archive.php

```diff
@@ -16,6 +16,7 @@ public static function extract($archive, $dir, $basename = '')
             if ($zip->open($archive)) {
                 $success = $zip->extractTo($tempdir);
                 $zip->close();
+                rex_set_rights::setRights($tempdir);
```

Comment:
Ist das hier nicht doppelt? (Oben rufst du direkt nach install_archive->extract() auch setRights auf?

Reply:
Hach, stimmt! Jetzt seh ich es auch.
Dann reicht es die Rechteänderung in der archive.php drin zu haben.